src/core/ExternalSensors.cpp

```diff
@@ -172,55 +190,63 @@ bool ExternalSensors::config() {
       if (json["sensorsNumber"] != NULL) {
         this->sensorsNumber = json["sensorsNumber"];
 
-        for (int i = 0; i < sensorsNumber; i++) {
+        sensor* sensors = new sensor[sensorsNumber];
```

Comment:
fomatting issue: this should be `sensor *sensors` (star character next to variable name). At least it's the LLVM way, even if we can argue having it next to the type makes more sense...